leapp/dialogs/components.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, key=None, label=None, description=None, default=None, reason=
         :param reason: The reason why we need this value.
         """
         self.key = key or type(self).key
-        self.label = label or type(self).label
+        self.label = label.replace('\n', '') if label else type(self).label
```

Comment:
replacing `Give me your\nhand` should not result in `Give me yourhand`
```suggestion
        self.label = label.replace('\n', ' ') if label else type(self).label
```

Reply:
I'd add a strip :D
```suggestion
        self.label = label.replace('\n', ' ').strip() if label else type(self).label
```

Reply:
Resolved for both label and description, thanks!